### DISCUSSION TITLE
Missing chia coins

### DISCUSSION THREAD

**Discussion by Sergey1314** [Chia-Network/chia-blockchain#16812] [Support]
Good afternoon, from my wallet installed on Windows Chia (address - xch1hsczc3vyjusp3lc9wxaqtz05hddjdldsvy9e2xwh5znhgymhzznqw46sm4 almost all my funds, in the amount of 6.17 coins, disappeared somewhere. Where did they go?

**Comment by BrandtH22** [comment]
Hey @Sergey1314 , as a heads up we can generally provide more timely and thorough support in our discord server (https://discord.gg/chia).

As for the missing funds, it appears that the coins were sent out of that address in September.
If that is a transaction that you did not perform then it is likely your keys or device have been compromised.

We recommend reviewing these best practices for securing your chia https://www.chia.net/2021/05/28/securing-your-chia-how-to-be-a-hard-target/ .

Unfortunately, since chia is a truly decentralized blockchain there is nothing we or anyone else would be able to do in order to return those funds to your wallet :/ If you did make that transaction and are just unable to see the funds in the new address we can help troubleshoot what syncing issue might be causing this.

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
